howto/install/deploy-terraform.md

```diff
@@ -0,0 +1,211 @@
+      }
+    ]
+
+Running `terraform plan -out=tfplan` after making this change will notify you that new resources need to be created. Carefully inspect the plan to verify the resource changes.
+Applying this plan in the same way as mentioned above i.e `terraform apply tfplan` will deploy the new Anbox Subcluster.
+
+## Enable HA Configuration
+
+The terraform plan comes with a recommended high availability (HA) configuration for Anbox Cloud deployments. This can be enabled by modifying the `deploy.tfvars` file as following:
+
+    ubuntu_pro_token = "<your_ubuntu_pro_token>"
+    constraints      = ["arch=arm64"]
+    anbox_channel    = "1.26/stable"
+    subclusters = [
+      {
+        name           = "a"
+        lxd_node_count = 1
+      }
+    ]
+    enable_ha = true // Deploy HA configuration for Anbox Cloud
+
+Running `terraform plan -out=tfplan` after making this change will notify you that new resources need to be created. Carefully inspect the plan to verify the resource changes.
+Applying this plan in the same way as mentioned above i.e `terraform apply tfplan` will deploy the new Anbox Subcluster.
+
+## Deploy AAR (Anbox Application Registry)
+
+You can deploy the Anbox Application Registry alongside Anbox Cloud by modifying the `deploy.tfvars` file created earlier, in the following manner:
+
+    ubuntu_pro_token = "<your_ubuntu_pro_token>"
+    constraints      = ["arch=arm64"]
+    anbox_channel    = "1.26/stable" // Channel to use for deploying Anbox Cloud
+    subclusters = [
+      {
+        name           = "a"
+        lxd_node_count = 5
+        registry_config = { // registry configuration for the subcluster
+            mode = "client"
+        }
+      }
+    ]
+    deploy_registry = true
+
+This will deploy a new Juju model dedicated to the Anbox Application Registry. Check the output of the `juju models` command to verify that a new model was created.
+
+```sh
+Model                Cloud/Region  Type  Status     Machines  Units  Access  Last connection
+anbox-controller     juju/default  lxd   available         4      4  admin   54 seconds ago
+anbox-registry       juju/default  lxd   available         1      1  admin   42 seconds ago
+anbox-subcluster-a*  juju/default  lxd   available         6      7  admin   36 seconds ago
+```
+
+## Add observability using COS (Canonical Observability Stack)
+
+You can enable observability for Anbox Cloud deployments by modifying the `deploy.tfvars` file created earlier, in the following manner:
+
+    ubuntu_pro_token = "<your_ubuntu_pro_token>"
+    constraints      = ["arch=arm64"]
+    anbox_channel    = "1.26/stable" // Channel to use for deploying Anbox Cloud
+    subclusters = [
+      {
+        name           = "a"
+        lxd_node_count = 1
+      }
+    ]
+    enable_cos = true
+
+Applying the plan using this configuration will deploy the required resources/charms to enable integration of Anbox Cloud components with COS (Canonical Observability Stack).
```

Comment:
We should provide example instructions on how this will look like and also explain what exactly the plan adds for COS support (e.g. grafana-agent and setting up relevant relations between grafana-agent and the Anbox Cloud components).